docs/docs/config/split.md

```diff
@@ -0,0 +1,86 @@
+
+:::
+
+| Config                                | Type   | Description                                                       | Default                                                       |
+| ------------------------------------- | ------ | ----------------------------------------------------------------- | ------------------------------------------------------------- |
+| `CONFIG_ZMK_SPLIT_WIRED`              | bool   | Use wired connection to communicate between split keyboard halves | y (if no BLE split and devicetree is set appropriately)       |
+| `ZMK_SPLIT_WIRED_UART_MODE`           | choice | The choice of which UART mode to use for TX/RX of data.           |                                                               |
+| `ZMK_SPLIT_WIRED_UART_MODE_ASYNC`     | bool   | Async (DMA) mode                                                  | y if the driver supports it (excluding nRF52 with known bugs) |
+| `ZMK_SPLIT_WIRED_UART_MODE_INTERRUPT` | bool   | Interrupt mode                                                    | `y` if the hardware supports it                               |
+| `ZMK_SPLIT_WIRED_UART_MODE_POLLING`   | bool   | Polling mode                                                      | `y` it neither other mode is supported                        |
+
+#### Async (DMA) Mode
+
+The following settings only apply when using wired split in async (DMA) mode:
+
+| Config                             | Type | Description                                                 | Default |
+| ---------------------------------- | ---- | ----------------------------------------------------------- | ------- |
+| `ZMK_SPLIT_WIRED_ASYNC_RX_TIMEOUT` | int  | RX Timeout (in microseconds) before reporting received data | 20      |
+
+#### Polling Mode
+
+The following settings only apply when using wired split in polling mode:
+
+| Config                              | Type | Description                                          | Default |
+| ----------------------------------- | ---- | ---------------------------------------------------- | ------- |
+| `ZMK_SPLIT_WIRED_POLLING_RX_PERIOD` | int  | Number of ticks between calls to poll for split data | 10      |
+
+## Devicetree
+
+### Wired Split
+
+Wired splits require a properly configured UART to function. If writing a shield, you may be able to use the standard UART already provided by the board, e.g. `&pro_micro_serial`. See shield overlays section in the [new shield guide](../development/hardware-integration/new-shield.mdx#shield-overlays) for details on the UART node labels provided by various interconnects. If creating your own board, or using custom pins for the UART, see the documentation on [pin control](../development/hardware-integration/pinctrl.mdx#additional-examples) to configure the pins for your UART.
```

Comment:
The first one should probably link to https://zmk.dev/docs/development/hardware-integration/pinctrl#predefined-nodes instead, which shows non-GPIO bus labels.

Reply:
Yeah, good call, forgot that had moved. Updated.